sdk/tables/Azure.Data.Tables/src/Queryable/ClientConvert.cs

```diff
@@ -0,0 +1,329 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Xml;
+
+namespace Azure.Data.Tables.Queryable
+{
+    internal static class ClientConvert
+    {
+        private const string SystemDataLinq = "System.Data.Linq, Version=" + FXAssembly.Version + ", Culture=neutral, PublicKeyToken=" + AssemblyRef.EcmaPublicKeyToken;
```

Comment:
What is this for?

Reply:
I think it may not be needed, but it seems to be a catch all for loading types we don't account for in `CreateKnownPrimitives`

Reply:
Can we remove it?